providers/from-env/pkg/provider.go

```diff
@@ -0,0 +1,141 @@
+package from_env
+
+import (
+	"github.com/open-feature/golang-sdk/pkg/openfeature"
+)
+
+// FromEnvProvider implements the FeatureProvider interface and provides functions for evaluating flags
+type FromEnvProvider struct {
+	envFetch envFetch
+}
+
+const (
```

Comment:
Can these errors come from the SDK?

Reply:
ive imported all available errors, and updated the style of the remaining errors / reasons to match